docs/setup/setting-up-site-analytics.md

```diff
@@ -49,20 +49,24 @@ following lines to `mkdocs.yml`:
 
     Besides page views and events, [site search] can be tracked to better
     understand how people use your documentation and what they expect to find.
-    In order to enable site search tracking, the following steps are required:
+    In order to enable site search tracking, the following steps are required
+    for Universal Analytics (UA-XXXXXXXX-X) properties:
 
     1.  Go to your Google Analytics __admin settings__
     2.  Select the property for the respective tracking code
     3.  Go to the __view settings__ tab
     4.  Scroll down and enable __site search settings__
     5.  Set the __query parameter__ to `q`
 
-    Note that currently, site search tracking is not supported with Google
-    Analytics 4 due to the more complicated manual setup. If you want to set up
-    site search tracking yourself, [this tutorial][tutorial] is a good start.
+    For Google Analytics 4 (G-XXXXXXXXXX) properties:
```

Comment:
Could you please move both guides into content tabs? Similar to the configuration section above the admonition 😊

Reply:
Sure thing

Reply:
<img width="889" alt="Screen Shot 2021-11-04 at 3 55 01 PM" src="https://user-images.githubusercontent.com/637005/140410544-d8787e23-9717-4c59-99f1-1504c8e1679a.png">

Reply:
Looks great!